src/core/toga/command.py

```diff
@@ -40,17 +42,20 @@ def __init__(self, action, label,
 
         self._enabled = self.action is not None
 
+        self._widgets = []
+
+        self.factory = get_platform_factory()
+        self._impl = self.factory.Command(interface=self)
```

Comment:
I'm not sure. At the moment we only load the icon for a command at the impl level.